server/xpub-model/entities/auditLog/index.js

```diff
@@ -0,0 +1,27 @@
+const BaseModel = require('@pubsweet/base-model')
+
+class AuditLog extends BaseModel {
+  static get tableName() {
+    return 'audit_log'
+  }
+
+  static get schema() {
+    return {
+      required: ['userId', 'action'],
+      properties: {
+        userId: { type: 'uuid' },
+        action: { type: 'string' },
+        objectId: { type: 'uuid' },
+        objectType: { type: ['string', 'null'] }
+      },
+    }
+  }
+
+  async delete() {
+    throw new Error('Unsupported operation')
```

Comment:
Nice! ... do we want one for `update` too?

Reply:
`delete` is provided by Pubsweet, which is why I took the opportunity to disable it. `save` is the only other method that is provided which has write access, other than the base methods provided by Objection.